app/controllers/events_controller.rb

```diff
@@ -4,7 +4,7 @@ class EventsController < ApplicationController
   before_action :check_authorization, only: [:edit, :update, :destroy]
 
   def index
-    @events = Event.all 
+    @events = Event.all.order('created_at DESC')
```

Comment:
This is OK direction, but keep in mind that for user more important is date when the event will happen not when it was created. Second thing: try to do that in more Rails way. Search in Rails documentation for the right syntax.

Reply:
I thought about started_at - but this 'the fresh one at the beginning' - make me think about created at ;)
```
def index
    @events = Event.all 
    @events.order('started_at DESC')
```

Or in the Event Model:
scope :desc, order(started_at: :desc)

??

Reply:
` order(started_at: :desc)` is good. Just prepare more descriptive name for this scope.

Reply:
scope :upcomming_events, order(started_at: :desc)

Will be ok?

Reply:
I was thinking about something longer, but telling exactly what it do like: `sort_by start_time`

Reply:
ok so should I put this part to index:     @events.order('started_at DESC')

AND
in the Event Model:
scope :desc, order(started_at: :desc) 

?

Reply:
No, I was saying about changing the name of this scope. I mean about `scope :sort_by start_time, order(started_at: :desc)` and you should use this scope instead of `@events.order('started_at DESC')`

Reply:
Ok please check it now :)

Reply:
> No, I was saying about changing the name of this scope. I mean about `scope :sort_by start_time, order(started_at: :desc)` and you should use this scope instead of `@events.order('started_at DESC')`

So in controller you should use `Event.sort_by_start_time` and in model `scope :sort_by start_time, order(started_at: :desc)`